contracts/contracts/RequestManager.sol

```diff
@@ -78,7 +78,8 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
         address targetAddress,
         uint256 amount,
         uint96 nonce,
-        uint32 validUntil
+        uint32 validUntil,
+        address indexed sender
```

Comment:
I'm thinking, since we already have `targetAddress`, shouldn't this be `sourceAddress`, to use terminology similar to the one already in the event?

Reply:
perfect idea